es/ae/oracle.js

```diff
@@ -61,27 +62,23 @@ async function getOracleObject (oracleId) {
  * @alias module:@aeternity/aepp-sdk/es/ae/oracle
  * @instance
  * @function
- * @category async
  * @param {String} oracleId Oracle public key
  * @param {Function} onQuery OnQuery callback
  * @param {Object} [options] Options object
  * @param {Number} [options.interval] Poll interval(default: 5000)
  * @return {Function} stopPolling - Stop polling function
  */
-async function pollForQueries (oracleId, onQuery, { interval = 5000 } = {}) {
-  const queries = (await this.getOracleQueries(oracleId)).oracleQueries || []
-  let quriesIds = queries.map(q => q.id)
-  await onQuery(queries)
-
-  async function pollQueries () {
-    const { oracleQueries: q } = await this.getOracleQueries(oracleId)
-    const newQueries = q.filter(({ id }) => !quriesIds.includes(id))
-    if (newQueries.length) {
-      quriesIds = [...quriesIds, ...newQueries.map(a => a.id)]
-      onQuery(newQueries)
-    }
+function pollForQueries (oracleId, onQuery, { interval = 5000 } = {}) {
+  const sentQueryIds = new Set()
+  const fetchQueries = async () => {
```

Comment:
fetch implies to me something is returned, but we just want the `onQuery` to be triggered

Reply:
I'm proposing a better naming in 5164358a931215ef629608c6d1369f302d966c75